### ISSUE TITLE
stand by, for platform-compat

### ISSUE THREAD

**Issue by kuboosoft** [UnitedRPMs/platform-compat#1]
platform-compat isn't necessary have the 'rpm' in united rpm; only maintain the github repo for testing/legacy purposes.

**Comment by sergiomb2** [comment]
Hi, I saw some of this "stand by" packages on repo , shouldn't be removed from there at least ? 
I mean https://googledrive.com/host/0ByOofIMiUfxPSG00M3hVRHVmYUE/x86_64/repoview/letter_j.group.html
https://googledrive.com/host/0ByOofIMiUfxPSG00M3hVRHVmYUE/x86_64/repoview/letter_p.group.html

**Comment by kuboosoft** [comment]
@sergiomb2  yes also  jsoncpp07 and  jsoncpp06 only el rpm; we only need wait; I reported the problem. https://bugzilla.redhat.com//show_bug.cgi?id=1336082

**Comment by sergiomb2** [comment]
Strange in my build.log it appears : 

-- Checking for module 'jsoncpp'
--   Found jsoncpp, version 0.6.0
-- Found JsonCpp: jsoncpp
